src/app/organizations/organization/organization.component.html

```diff
@@ -28,8 +28,8 @@
     </mat-card>
     <mat-card fxFlex class="my-3">
       <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="10px">
-        <div>
-          <img src="https://via.placeholder.com/150" alt="Logo" class="img-circle">
+        <div >
+          <img [src]="org.avatarUrl != null ? gravatarUrl : '../../../assets/images/dockstore/PlaceholderLC.png'" class="img-circle" height="150" width="150">
```

Comment:
I would probably have `gravatarUrl || '../.....png`. This code assumes that if avatarUrl is not null, then gravatarUrl is not as well. They probably always are in sync, but just test against gravatarUrl itself. Assuming gravatarUrl is not set if avatarUrl isn't set.